wintermotes/week5/ass6/app/components/fields.js

```diff
@@ -0,0 +1,220 @@
+var React = require('react');
+var List = require('./list');
+var Messages = require('./message').Messages;
+var Message = require('./message').Message; 
+var archivedMessages = require('./message').ArchivedMessages;
+var ArchivedMessage = require('./message').ArchivedMessage;
+
+var ListStore = require('../stores/list-store');
+var ListActions = require('../actions/list-actions');
+
+var MessageStore = require('../stores/message-store');
+var MessageActions = require('../actions/message-actions');
+
+import NotificationBar from '../components/notification-bar'
+
+function getLists(){
+	return ListStore.getAllLists();
+}
+
+function getNotifications(){
+	return MessageStore.getNotifications()
+}
+var CreateListField = React.createClass({
```

Comment:
I don't like the naming `Create..`here. The var is containing the final component so no need for the naming